docs/continuous_integration.rst

```diff
@@ -1,11 +1,17 @@
 Continuous Integration
 ======================
```

Comment:
I don't like the way this top-level CI page is structured right now. It makes the section about MetaCI look like the most important thing to read about CI, when it's actually more of a side note about one particular system you might use to run CumulusCI for CI purposes.

I think we need an overall introduction about the topic of CI: defining what it is, emphasizing how CumulusCI makes it possible to run the same automation that is used by devs/QEs/customers within the context of a CI system, and explaining how there are various different systems you could choose for running CI (i.e. github actions, MetaCI, CircleCI, others...)

Then link to the other pages about CumulusCI Flow and github actions, but I think I would write a paragraph about each one to contextualize what info is there and who would be interested in it, rather than just listing out their outlines in a ToC.